### PR TITLE
feat(files-search): implement advanced search options

### PR DESCRIPTION
**Pull request by nathan-power**
**What this PR does** 📖
- Create UI for search buttons in input component 
- Provide option to search through all directories 
- Add text displayed when it's searching specifying what user is searching for and in which directories
- Add UiResultsMessage component and apply for empty searches

**Which issue(s) this PR fixes** 🔨
AP-2097
AP-2162

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: pages/files/Files.html [hunk 1]
```diff
@@ -4,13 +4,23 @@
...
+
     <FilesControlsSwitch />
   </div>
   <div class="files-view">
-    <TypographyText v-if="!directory.length">
-      {{ $t('pages.files.empty') }}
-    </TypographyText>
+    <div v-if="!directory.length">
+      <UiResultsMessage v-if="!searchValue" :title="$t('pages.files.empty')" />
+      <UiResultsMessage v-else :title="$t('pages.files.search.empty')" />
```
Comment (josephmcg) [inline pages/files/Files.html:22]: Since its the same component, you can ternary check and pass a different string as title prop

**Review by josephmcg — COMMENTED** [review]
Just saw 1 thing, will take a closer look when i get back (unless its already merged 😁)

**Review by aewing — APPROVED** [review]

File: pages/files/Files.html (hunk 1 above)
Comment (nathan-power) [reply pages/files/Files.html:22]: You're right, best to keep it to one element

**Comment by phillsatellite** [comment]
@WanderingHogan these are the two things I noticed on this Branch

https://user-images.githubusercontent.com/93608357/185005803-bc8f38d6-9d94-4ab8-a5ce-0b9955bea44f.mov

**Pull request merged**